include/swift/AST/Decl.h

```diff
@@ -3891,7 +3891,7 @@ class NominalTypeDecl : public GenericTypeDecl, public IterableDeclContext {
   /// protocols to which the nominal type conforms. Furthermore, the resulting
   /// set of declarations has not been filtered for visibility, nor have
   /// overridden declarations been removed.
-  TinyPtrVector<ValueDecl *> lookupDirect(DeclName name,
+  TinyPtrVector<ValueDecl *> lookupDirect(DeclName name, SourceLoc loc = SourceLoc(),
```

Comment:
Should we plan to take away that default argument at some point, once we've managed to update all of the callers?